### ISSUE TITLE
Error finding parent and previousFrame

### ISSUE THREAD

**Issue by jameswzhu** [stack-of-tasks/pinocchio#1948]
Hello, I have a fresh install of the pinocchio3-preview branch. I am trying to run the `simulation-contact-dynamics.py` example, and as I build the model from the urdf, I find that at line 66, `contact_model = pin.RigidConstraintModel(pin.ContactType.CONTACT_6D,frame.parent,frame.placement)`, the `frame.parent` and `frame.previousFrame` contain the error: `Traceback (most recent call last):
  File "<string>", line 1, in <module>
Boost.Python.ArgumentError: Python argument types in
    None.None(Frame)
did not match C++ signature:
    None(pinocchio::ModelItem<pinocchio::FrameTpl<double, 0> > {lvalue})`. Let me know if I have done something wrong with the installation here? Thanks!

**Comment by jcarpent** [comment]
Could you provide the steps to reproduce your issue? Could you provide a docker image?

**Comment by jameswzhu** [comment]
I uploaded the docker image here: [(https://hub.docker.com/r/zhuj1198/pinocchio)]. Thank you!

**Comment by jameswzhu** [comment]
Hello, have you had a chance to take a look at this?

**Comment by jcarpent** [comment]
@MegMll Could you check this issue?

**Comment by MegMll** [comment]
Hello, 

I checked your installation in the docker and it seems that you compiled pinocchio without collision support, which is necessary for contact dynamics.
So I would suggest, you rebuild pinocchio by adding collision support (with `-DBUILD_WITH_COLLISION_SUPPORT=ON`) which requires you to first compile the hpp-fcl library.
I tried to compile it in your docker, but it seems that you have some missing dependencies.
